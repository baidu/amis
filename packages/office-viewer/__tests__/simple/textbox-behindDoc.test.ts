import {snapShotTest} from '../snapShotTest';

test('textbox-behindDoc', async () => {
  snapShotTest('./docx/simple/textbox-behindDoc.xml');
});
