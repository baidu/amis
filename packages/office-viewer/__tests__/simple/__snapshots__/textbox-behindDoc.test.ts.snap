// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`textbox-behindDoc 1`] = `
<div
  class="docx-viewer-0 docx-viewer-wrapper"
  id="root"
>
  <style>
    
  
  .docx-viewer-wrapper {

  }

  .docx-viewer-wrapper &gt; article &gt; section {
    background: white;
  }

  /** docDefaults **/

  .docx-viewer-0 p {
    margin: 0;
    padding: 0;
    line-height: 1.5;
  }

  .docx-viewer-0 table {
    border-spacing: 0;
  }

  .docx-viewer-0 .docx-viewer-0-p {
    
  }

  .docx-viewer-0 .docx-viewer-0-r {
    white-space: pre-wrap;
    overflow-wrap: break-word;
    font-family: var(--docx-theme-font-minorHAnsi), var(--docx-theme-font-minorEastAsia);
font-size: 16.00px;

  }
  

  
    
      .docx-viewer-0 .docx-viewer-0-Normal {
        text-align: justify;

      }
      
    
    
    
    
    
    
    
    
    
    
    
    
 .docx-viewer-0 .docx-viewer-0-TableNormal {
  border-collapse: collapse;
  margin-left: 0.00px;

 }

 .docx-viewer-0 .docx-viewer-0-TableNormal &gt; tbody &gt; tr &gt; td {
  padding-top: 0.00px;
padding-left: 7.20px;
padding-bottom: 0.00px;
padding-right: 7.20px;

 }
 
    
    
    
    
    
    
    
  
  </style>
  <style>
    /** embedded fonts **/
  </style>
  <article>
    <section
      style="position: relative; width: 815.98px; padding: 96.00px 96.00px 96.00px 96.00px;"
    >
      <p
        class="docx-viewer-0-p"
      >
        <span
          class="docx-viewer-0-r"
        >
          <div
            style="z-index: 251659264; position: absolute; left: 144.00px; top: 77.80px; width: 122.79px; height: 53.58px; column-count: 1; background-color: rgb(255, 255, 255); border-width: 0.67px; border-style: solid;"
          >
            <p
              class="docx-viewer-0-p"
            >
              <span
                class="docx-viewer-0-r"
              >
                t
              </span>
              <span
                class="docx-viewer-0-r"
              >
                ext
              </span>
            </p>
          </div>
        </span>
        <span
          class="docx-viewer-0-r"
        >
          <div
            style="z-index: 251658239; position: absolute; left: 122.79px; top: 64.41px; width: 164.09px; height: 84.84px; background-color: rgb(112, 173, 71); column-count: 1; border-style: solid;"
          >
            <p
              class="docx-viewer-0-p"
            >
              Â 
            </p>
          </div>
        </span>
      </p>
    </section>
    <section
      style="position: relative;"
    />
  </article>
</div>
`;
